sdks/java/io/hadoop-format/src/main/java/org/apache/beam/sdk/io/hadoop/format/HadoopFormatIO.java

```diff
@@ -702,6 +702,11 @@ public void populateDisplayData(DisplayData.Builder builder) {
                     "mapreduce.job.inputformat.class",
                     hadoopConfig.get("mapreduce.job.inputformat.class"))
                 .withLabel("InputFormat Class"));
+        builder.addIfNotNull(
+            DisplayData.item(
+                    "mapreduce.input.fileinputformat.inputdir",
+                    hadoopConfig.get("mapreduce.input.fileinputformat.inputdir"))
```

Comment:
I'm a little unsure if we should add a safeguarding string-truncation here: technically Hadoop allows you to set a [list of comma-separated paths](https://github.com/apache/hadoop/blob/207c92753fc4782ebab3995eb8e4a2a62c744f27/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/lib/input/FileInputFormat.java#L524) as input, so this could potentially be very long. Maybe we can clip it at 250 chars?

Reply:
Yeah, sounds good to me